File: Modul 1 Sem 1/Phython/PRAK1010-2410817110005-MuhammadRizqanKarim.py
from math import sqrt

height = 12
floor = 5

print(f"Diketahui: ")
print(f"Alas = {floor} cm")
print(f"Tinggi = {height} cm")

hypotenuse = sqrt((height * height) + (floor * floor))
perimeter = height + floor + hypotenuse
area = 0.5 * floor * height

print("")
print("Jawaban :")
print(f"Sisi A = {height} cm")
print(f"Sisi B = {hypotenuse:.0f} cm")
print(f"Sisi C = {floor} cm")
print(f"Keliling = {perimeter:.0f} cm")
print(f"Luas = {area:.0f} cm")

File: Modul 1 Sem 1/C/PRAK1010-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>
#include <math.h>
int main()
{
    int area, perimeter, hypotenuse, height = 12, floor = 5;

    printf("Diketahui:\n");
    printf("Alas = %d cm\n", floor);
    printf("Tinggi = %d cm\n", height);
    printf("\n");

    hypotenuse = sqrt((height * height) + (floor * floor));
    perimeter = height + floor + hypotenuse;
    area = 0.5 * floor * height;

    printf("Jawaban :\n");
    printf("Sisi A = %d cm\n", height);
    printf("Sisi B = %d cm\n", hypotenuse);
    printf("Sisi C = %d cm\n", floor);
    printf("Keliling = %d cm\n", perimeter);
    printf("Luas = %d cm", area);

    return 0;
}
